source/collections.txt

```diff
@@ -44,20 +53,35 @@ information for each collection in the selected database:
   collection. Hover over a :guilabel:`Collation` banner to view
   the properties for that collection.
 
-.. figure:: /images/compass/collection-tab2.png
-   :figwidth: 720px
+.. _compass-views:
+
+Views
+~~~~~
+
+Views are read-only results of an aggregation run against a collection. 
```

Comment:
Should we also explain the use case for views? Not sure how much we want to get into fundamentals here.

Reply:
I like the idea but maybe would be better on the main Views page? I'll add a sentence there.